Detector/DetCRD/compact/CRD_common_v01/DC_Simple_v01_01.xml

```diff
@@ -44,6 +44,10 @@
     <constant name="DC_Endcap_rmax" value="SDT_radius_max"/>
     <constant name="DC_Endcap_dz" value="DC_Endcap_z"/>
 
+    <constant name="DC_inner_chamber_enabled" value="0"/>
+    <constant name="DC_outer_chamber_enabled" value="0"/>
+    <constant name="DC_all_chamber_enabled" value="1"/>
```

Comment:
I think the first two variables are enough, the third one is not necessary.

Reply:
I have changed the control parameter to oneChange the control parameter to one, please check it, thank you.